src/FluentSchema.integration.test.js

```diff
@@ -224,7 +224,9 @@ describe('S', () => {
   })
 
   describe('compose keywords', () => {
-    const ajv = new Ajv()
+    const ajv = new Ajv({
+      allowUnionTypes: true
```

Comment:
Why this change?

Reply:
Because ajv will warn and spam the log output, because here type will be integer and string.

https://ajv.js.org/strict-mode.html#union-types